src/main/java/io/a2a/spec/UnsupportedOperationError.java

```diff
@@ -0,0 +1,8 @@
+package io.a2a.spec;
+
+public class UnsupportedOperationError extends JSONRPCError {
+
+    public UnsupportedOperationError(int code, String message, Object data) {
```

Comment:
For classes like this that extend `JSONRPCError`, I believe we'll need the `@JsonCreator` annotation on the constructor and `@JsonProperty("blah")` annotations on each parameter to be able to successfully create the object from JSON. Let me know if I'm missing something though.

Reply:
I need to read up a bit on this :-)